### PR TITLE
chore: update cheatsheet for new provider syntax

### PR DESCRIPTION
**Pull request by Foxandxss**
**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit-message-format
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Other... Please describe:

**Other information**:

This updates the cheatsheet for RC2, we need this PR to generate the new live site at angular.io.

/cc @naomiblack

### CODE REVIEW

**Comment by vicb** [comment]
The `{}` syntax does not work with Dart

**Comment by Foxandxss** [comment]
We are aware of that Victor, @kwalrath gave me green light. One of our team members will update all the cheatsheet for dart soon.

The old syntax that is actually in the cheatsheet doesn't work either.

File: modules/@angular/docs/cheatsheet/bootstrapping.md [hunk 1]
```diff
@@ -8,11 +8,11 @@ Bootstrapping
 
 @cheatsheetItem
 syntax(ts dart):
```
Comment (vicb) [inline modules/@angular/docs/cheatsheet/bootstrapping.md]: I mean that

Comment (Foxandxss) [inline modules/@angular/docs/cheatsheet/bootstrapping.md]: yes, but that was like that before, giving a wrong example for dart. So it wasn't good before, it is not good now either.

I am updating this for typescript convenience. The dart effort will come later.

**Comment by kwalrath** [comment]
@chalin: making sure you see this.

**Pull request merged**